content/en/docs/gke/deploy/deploy-cli.md

```diff
@@ -171,21 +171,22 @@ gcloud.compute.zone | The zone to use for zonal resources; must be in gcloud.com
 
 To deploy kubeflow, run the following command:
 
-   ```
-   make apply
-   ```
+```
+make apply
+```
 
-   * If resources can't be created because `webhook.cert-manager.io` is unavailable wait and
-     then rerun `make apply`
+* If resources can't be created because `webhook.cert-manager.io` is unavailable wait and
+  then rerun `make apply`
 
-     * This issue is being tracked in [kubeflow/manifests#1234](https://github.com/kubeflow/manifests/issues/1234)
+  * This issue is being tracked in [kubeflow/manifests#1234](https://github.com/kubeflow/manifests/issues/1234)
 
-   * If resources can't be created with error message like:
-    ```
-    error: unable to recognize ".build/application/app.k8s.io_v1beta1_application_application-controller-kubeflow.yaml": no matches for kind "Application" in version "app.k8s.io/v1beta1”
-    ```
+* If resources can't be created with an error message like:
+
+  ```
+  error: unable to recognize ".build/application/app.k8s.io_v1beta1_application_application-controller-kubeflow.yaml": no matches for kind "Application" in version "app.k8s.io/v1beta1”
+  ```
 
-    This is expected race condition and you can try rerun `make apply` again, because Kubernetes API server hasn't been ready for CRDs when their instances were applied. This can happen multiple times for different resource kinds.
+  This issue occurs when the CRD endpoint hadn't been established in Kubernetes API server when the CRD's custom object was applied. This issue can happen multiple times for different kinds of resource. To resolve this issue, try running `make apply` again.
```

Comment:
Suggestion:

- Tenses are hard and here you've got _present_, _past perfect_ and _past_ (in passive) in one sentence.
  - One way to make it simpler for the reader is to use present tense most of the time e.g.:

```diff
- This issue occurs when the CRD endpoint hadn't been established in Kubernetes API server 
+ This issue occurs when the CRD endpoint isn't established in the Kubernetes API server  
- when the CRD's custom object was applied. This issue can happen multiple times for 
+ when the CRD's custom object is applied. This issue can happen multiple times for 
different kinds of resource. To resolve this issue, try running `make apply` again.
```

Reply:
Thanks! It was hard for me too